src/index.ts

```diff
@@ -26,13 +26,19 @@ function loader(this: interfaces.Webpack, contents: string) {
         return;
     }
 
+    // Load files generated by previous loaders in the same chain
+    const generatedFiles = getGeneratedFiles(this);
```

Comment:
Can we put the new functionality behind a loader option please?  Something like `usePreviousLoaderGeneratedFiles`.  (Maybe something shorter?)  I don't want this to run unless people have "flicked the switch".

So calling `getGeneratedFiles` etc only happens when you're using `usePreviousLoaderGeneratedFiles`